test/record_test.rb

```diff
@@ -365,4 +371,20 @@ def test_spf_is_deprecated
     Record::SPF.new(fqdn: 'dns-test.shopify.io.', txtdata: 'v=spf1 -all', ttl: 3600)
     assert_includes($stderr.string, "SPF record type is deprecated (See RFC 7208 Section 14.1)")
   end
+
+  def test_valid_ptr_record
+    assert_predicate(Record::PTR.new(
+      fqdn: '4.0.41.198.in-addr.arpa',
```

Comment:
I don't know the spec for PTR offhand, but should we validate that it's only up to (or maybe exactly - really not sure) four octets, and that they are numbers in the range 0-255?

Reply:
Had to take a second look over [RFC 1035 § 3.5](https://tools.ietf.org/html/rfc1035):

> Domain names in the IN-ADDR.ARPA domain are defined to have up to four labels in addition to the IN-ADDR.ARPA suffix.  Each label represents one octet of an Internet address, and is expressed as a character string for a decimal value in the range 0-255 (with leading zeros omitted except in the case of a zero octet which is represented by a single zero).

I can write up a validation for this.